### PR TITLE
Add multiple contributors to the credits

### PR DESCRIPTION
**Pull request by arthuro555**
Adds multiple missing contributors to the list.

### CODE REVIEW

**Comment by Bouh** [comment]
Can you sort by alphabetical order? Thx

File: newIDE/app/src/MainFrame/AboutDialog.js [hunk 1]
```diff
@@ -39,16 +39,23 @@ const styles = {
 // If you contributed to GDevelop but you're not in the list, please
 // send a Pull Request on GitHub or open an issue ;)
 const contributors = [
+  // Core team
+  {
+    name: 'Clément Pasteau',
+    description: 'CTO of the core development team',
```
Comment (4ian) [inline newIDE/app/src/MainFrame/AboutDialog.js:45]: Uber nitpicking: Maybe just "Core development team"? (Avoiding titles in this list and focusing on the tasks done)

**Review by 4ian — APPROVED** [review]

**Comment by 4ian** [comment]
Thanks for taking the time to do this :)

File: newIDE/app/src/MainFrame/AboutDialog.js [hunk 2]
```diff
@@ -60,13 +67,12 @@ const contributors = [
...
     description:
-      'Numerous contributions to GDevelop (Video extension, advanced shapes, bugfixes, new conditions, examples...)',
+      'Numerous code contributions to GDevelop and community management',
     link: 'https://www.witly.fr',
   },
   {
     name: 'Todor Imreorov',
-    description:
-      'Numerous contributions to GDevelop, added Piskel sprite editor and jfxr',
+    description: 'Numerous code contributions to GDevelop',
```
Comment (blurymind) [inline newIDE/app/src/MainFrame/AboutDialog.js:75]: how come theres no link to my github profile? :)

**Pull request merged**